src/guides/payment-flows.md

```diff
@@ -55,26 +55,27 @@ barcode or QR code displayed on the customer's smart device.
 
 1. Customer generates a short-lived [Patron Code][] via Centrapay API and displays
    it as a barcode on their smart device
-2. Merchant scans the barcode, optionally [retrieves the Patron Code][] for additional
-   information on the customer and then [creates a Payment Request][] via Centrapay API,
-   with the Patron Code as a parameter.
+2. Merchant scans the barcode, optionally checks for support using the
```

Comment:
Ditto, can we point them to the specific endpoint they should use to check for support?